### PR TITLE
Live Analog Clock using JavaScript #2181

### LINKED ISSUE #2181: Live Analog Clock using JavaScript
Respected Mentors, I have made a live **Analog Clock** using HTML, CSS and JavaScript.

Considering your wonderful repository of projects, I would like to contribute to **DevGeeks** this mini-project under the JS-Projects folder.

Being a newbie open-source contributor, I would humbly request @pranjay-poddar and other fellow mentors to kindly assign this issue to me to be able to make a successful and worthy contribution to DevGeeks as a GSSoC'23 Contributor.
image

### PR DESCRIPTION
**Pull request by ahanban15**
Live Analog Clock using JavaScript #2181

Close #2181 

# Checklist:

- [X] I have mentioned the issue number in my Pull Request.
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have created a helpful and easy to understand `README.md`

### CODE REVIEW

**Comment by ahanban15** [comment]
@pranjay-poddar and other mentors, I have contributed the required Live Analog clock project using JS under JS-Projects folder. Kindly judge my contribution and please suggest any changes if necessary. Kindly provide the approving review to be able to merge my contribution to DevGeeks for a humble successful contribution under GSSoC'23.

**Comment by ahanban15** [comment]
@pranjay-poddar and other mentors, I have contributed the required Live Analog clock project. Kindly judge my contribution and please suggest any changes if necessary. Kindly provide the approving review to be able to merge my contribution to DevGeeks for a humble successful contribution

**Review by pranjay-poddar — CHANGES_REQUESTED** [review]
Remove.vscode folder committed in root from your commit

**Comment by ahanban15** [comment]
Kindly review the new commit made @pranjay-poddar and other mentors. Removed the .vscode folder as suggested.

**Review by MohitGupta121 — CHANGES_REQUESTED** [review]
@ahanban15 please remove the image from PR and upload it somewhere in cloud and use the link of the image to load the image into your Project.

**Comment by ahanban15** [comment]
Issue #2260 
@pranjay-poddar @MohitGupta121 done as suggested, kindly review the PR and merge my contribution to DevGeeks for a humble successful contribution under GSSoC'23.

File: Js-Projects/Live Analog Clock using JS/clock.jpg
Comment (MohitGupta121) [inline Js-Projects/Live Analog Clock using JS/clock.jpg:1]: @ahanban15 image is still there

**Review by MohitGupta121 — CHANGES_REQUESTED** [review]

File: Js-Projects/Live Analog Clock using JS/clock.jpg
Comment (ahanban15) [reply Js-Projects/Live Analog Clock using JS/clock.jpg:1]: Done sir, kindly verify now.

File: .vscode/settings.json
Comment (MohitGupta121) [inline .vscode/settings.json:1]: @ahanban15 please not change this file

**Pull request closed without merging**